libs/shared/contentful/src/lib/contentful.client.ts

```diff
@@ -58,33 +64,36 @@ export class ContentfulClient {
     return result;
   }
 
+  @FirestoreCacheable(
+    {
+      cacheKey: (args: any) => cacheKeyForQuery(args[0], args[1]),
+      strategy: new NetworkFirstStrategy(),
+      ttlSeconds: DEFAULT_FIRESTORE_CACHE_TTL,
+    },
+    CONTENTFUL_QUERY_CACHE_KEY
```

Comment:
Unfortunately this can't come from DI (Contentful config) since it's provided to the decorator and won't exist in time.